source/extensions/filters/http/aws_lambda/aws_lambda_filter.cc

```diff
@@ -392,6 +405,10 @@ void AWSLambdaFilter::lambdafy() {
 void AWSLambdaFilter::handleDefaultBody() {
   if ((!has_body_) && function_on_route_->defaultBody()) {
     Buffer::OwnedImpl data(function_on_route_->defaultBody().value());
+    if (isRequestTransformationNeeded()) {
+      decoder_callbacks_->addDecodedData(data, true);
+      transformRequest();
+    }
```

Comment:
if the request is transformed here, would the wrong sha be computed below?
please cover by tests if needed?

Reply:
Clarified this here: https://github.com/solo-io/envoy-gloo/pull/177/commits/5ade735620c3735a894bcb4f5765d4b64e118132
now there is only a single call to `transformRequest` in the lambda filter, in `lambdafy`, after the body has potentially been replaced by the default value